src/azul/plugins/metadata/hca/indexer/transform.py

```diff
@@ -1566,7 +1566,7 @@ class BundleAsEntity(DatedEntity):
     def __init__(self, bundle: api.Bundle) -> None:
         super().__init__()
         self.document_id = bundle.uuid
-        date = parse_dcp2_datetime(bundle.version)
+        date = parse_dcp2_version(bundle.version)
```

Comment:
This can't be right. The version format depends on the repository plugin while you are hard-coding it here in a metadata plugin.